src/node_url.cc

```diff
@@ -552,6 +552,19 @@ static inline bool IsSpecial(std::string scheme) {
   return false;
 }
 
+// https://url.spec.whatwg.org/#start-with-a-windows-drive-letter
```

Comment:
nit: add a `https://`

Reply:
I think I'm missing something:

`// https://url.spec.whatwg.org/#start-with-a-windows-drive-letter`

Does have `https` misread?

Reply:
Oh I'm sorry. It's probably a Chrome extension I'm using.